releasenotes/notes/readout-mitigation-classes-2ef175e232d791ae.yaml

```diff
@@ -0,0 +1,8 @@
+---
+features:
```

Comment:
You should add a bullet point release note here documenting the addition of the abstract class and how it's used to build new mitigators with a standard api. Also, the api additions to existing data structures: `Counts.shots()` and QuasiDistribution's stddev_upper_bound stuff need separate feature bullet points here.

For this note itself you really should document how and why a user would use `CorrelatedReadoutMitigator` or `LocalReadoutMitigator` too